### PR TITLE
feat(plasma-typo): Add bold type for `Dspl` and `H` components

### PR DESCRIPTION
**Pull request by neretin-trike**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.67.0-canary.7.2311997036.0
  npm install @salutejs/plasma-core@1.59.0-canary.7.2311997036.0
  npm install @salutejs/plasma-temple@1.68.0-canary.7.2311997036.0
  npm install @salutejs/plasma-typo@0.6.0-canary.7.2311997036.0
  npm install @salutejs/plasma-ui@1.104.0-canary.7.2311997036.0
  npm install @salutejs/plasma-web@1.102.0-canary.7.2311997036.0
  npm install @salutejs/plasma-cy-utils@0.12.0-canary.7.2311997036.0
  # or 
  yarn add @salutejs/plasma-b2c@1.67.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-core@1.59.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-temple@1.68.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-typo@0.6.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-ui@1.104.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-web@1.102.0-canary.7.2311997036.0
  yarn add @salutejs/plasma-cy-utils@0.12.0-canary.7.2311997036.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-7/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-7/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-7/

**Review by LamaEats — APPROVED** [review]

File: packages/plasma-typo/src/components/Dspl.tsx [hunk 1]
```diff
@@ -2,20 +2,21 @@ import styled from 'styled-components';
...
-import { dsplL, dsplM, dsplS } from '../tokens';
+import { dsplL, dsplLBold, dsplM, dsplMBold, dsplS, dsplSBold } from '../tokens';
+import type { BoldProps } from '../types';
 
-export const DsplL = styled.div<SpacingProps & BreakWordProps>`
+export const DsplL = styled.div<SpacingProps & BoldProps & BreakWordProps>`
     ${applyHyphens}
     ${applySpacing}
-    ${dsplL}
+    ${({ bold = true }) => (bold ? dsplLBold : dsplL)}
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply packages/plasma-typo/src/components/Dspl.tsx:11]: предлагаешь завести новый миксин?

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-7/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-7/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-7/
  ui storybook: http://plasma.sberdevices.ru/pr-7/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-7/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-7/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-7/temple-storybook/

**Pull request merged**